Readme.md

```diff
@@ -1,628 +1,190 @@
-
-[Specific CSS overrides]: http://jsfiddle.net/gh/get/mootools/1.5.2/TIBCOSoftware/JS-visualize/tree/master/inputControl-css/specific-overrides/
-[all types]: http://jsfiddle.net/gh/get/mootools/1.5.2/TIBCOSoftware/JS-visualize/tree/master/inputControl-css/all-types/
-
+### Add new jsfiddle sample
+- add a subfolder under necessary samples group (`adhoc view`, `dashboard`, `inputControl`, `report` etc. 
+  see [Folders structure](#folders-structure))
+- add `demo.*` files with actual sample content to this folder
+- integrate this sample into the documentation itself: see [How to update documentation](#how-to-update-documentation)
 
+## Slatedocs
+[slatedocs][slatedocs] is used to generate [online documentation][online documentation]
+All files related to `slatedocs` are placed under `slatedocs` folder.
 
-License
-=================
+### Slatetdocs folder structure
+```
+|-lib
+|-source
+|  |-fonts
+|  |-images
+|  |-includes
+|  |-javascripts
+|  |-layout
+|  |-stylesheets
+|  |-index.html.md
+|-*.*
+```
 
-Copyright © 2018 TIBCO Software Inc. All Rights Reserved. 
+`index.html.md` - this is the actual file which should be modified in order to update documentation.
+In most cases you do not have to change any other files in the `slatedocs` folder
+
+### Build docs
+The simplest way to build documentation is to use `docker`:
+- install [docker][docker]
+- execute 
+  ```shell
+    cd ./slatedocs
+    docker run --rm --name slate -v $(pwd)/build:/srv/slate/build -v $(pwd)/source:/srv/slate/source slatedocs/slate
+  ```
+  Build result will be generated to `slatedocs/build` folder.
+- Now open this file `slatedocs/build/index.html` - you should see your documentation 
+
+For other ways to build documentation - see slate [getting started][slatedocs getting started] docs
+
+### Start dev server
+The simplest way to build documentation is to use `docker`:
+- install [docker][docker]
+- execute
+  ```shell
+    cd ./slatedocs
+    docker run --rm --name slate -p 4567:4567 -v $(pwd)/source:/srv/slate/source slatedocs/slate serve
+  ```
+- Now open [http://127.0.0.1:4567](http://127.0.0.1:4567) - you should see documentation in your browser
+- Every time you will do changes in the `slatedocs/sources` folder - changes will be available in the browser after 
+  page refresh
+
+For other ways to build documentation - see slate [getting started][slatedocs getting started] docs
+
+### Update slatedocs template to the latest version
+Template is slightly changed but generally it should be easy to update slate template to the latest version
+if necessary. To update template:
+- clone original [slatedocs repo][slatedocs]
+- copy all files and folders from the cloned repo except `.github` and `source/index.html.md` to `slatedocs` 
+  folder and replace files if they exists
+- copy `.github` folder from cloned the repo to the root `.github` folder and replace files if they exists.
```

Comment:
Change "folder from cloned the repo to the root" to "folder from the cloned repo to the root"

Reply:
fixed